<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseMode</class>
 <widget class="QDialog" name="ChooseMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>381</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="btnLocalGame">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>LocalGame</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNetworkGame">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>NetworkGame</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQuit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>as client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>as server</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnCancelConnect">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEnterConnect">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelConnection">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>as client and BLACK team.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTextEdit" name="ConnectIPtextEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="ConnectPorttextEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClientEnterConnect">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClientCancel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelClientIP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelClientPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect Port</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="labelServerIP">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="ServerPorttextEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnServerEnter">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Waiting</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnServerCancel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelServerPort">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
     <property name="buddy">
      <cstring>ServerPorttextEdit</cstring>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
